lib/player.js

```diff
@@ -2316,12 +2316,12 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       },
       onManifestUpdated: () => {
         const eventName = shaka.util.FakeEvent.EventName.ManifestUpdated;
-        const data = (new Map()).set('isLive', this.isLive());
+        const data = (new Map()).set('isLive', this.isDynamic());
```

Comment:
This is breaking change.
It should stay like it was, additionally you may add new `isDynamic` field to this event.